plansys2_pddl_parser/src/plansys2_pddl_parser/Utils.cpp

```diff
@@ -884,12 +884,38 @@ std::vector<uint32_t> getSubtrees(const plansys2_msgs::msg::Tree & tree)
       }
 
     default:
-      std::cerr << "getSubtrees: Error parsing expresion [" << toString(tree) << "]" << std::endl;
+      std::cerr << "getSubtreeIds: Error parsing expresion [" << toString(tree) << "]" << std::endl;
   }
 
   return {};
 }
 
+std::vector<plansys2_msgs::msg::Tree> getSubtrees(const plansys2_msgs::msg::Tree & tree)
+{
+  std::vector<uint32_t> node_ids = parser::pddl::getSubtreeIds(tree);
+  std::vector<plansys2_msgs::msg::Tree> subtrees;
+  for (auto node_id : node_ids)
+  {
```

Comment:
This should be in the line above... It's strange that linter didn't say anything here :thinking: